packages/jaeger-ui-components/src/TracePageHeader/TracePageSearchBar.tsx

```diff
@@ -24,11 +23,24 @@ import UiFindInput from '../common/UiFindInput';
 import { ubFlexAuto, ubJustifyEnd } from '../uberUtilityStyles';
 // eslint-disable-next-line no-duplicate-imports
 import { memo } from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
 
-export const getStyles = () => {
+export const getStyles = (theme: GrafanaTheme2) => {
   return {
     TracePageSearchBar: css`
       label: TracePageSearchBar;
+      float: right;
+      position: sticky;
+      top: 8px;
+      right: 0;
+      z-index: 5;
+      background: ${theme.colors.background.primary};
+      margin-top: 8px;
```

Comment:
Kind of hacky, but I wanted to add some margin between search bar and the container - so I added padding + margin to give spacing. Also added box-shadow/border-radius to distinguish the search bar a bit more. 

Would be cool in the future to detect when it's scrolled and apply the box-shadow then.